quickwit/quickwit-indexing/src/actors/indexing_service.rs

```diff
@@ -572,10 +723,13 @@ mod tests {
         );
         let (indexing_server_mailbox, indexing_server_handle) =
             universe.spawn_actor(indexing_server).spawn();
-        let observation = indexing_server_handle.observe().await;
```

Comment:
On my machine, this test takes 6-7s to run on `main` vs. 9-10s. Why is that, it's not like we are doing more CPU-heavy work. I think that is why you had to switch from `observe` to `process_pending_and_observe`.

Reply:
I changed the way to test, but having to `simulate_time_shift` and wait for the supervise message to be handled seems to be the cause. reducing the assertions reduces the time it takes.